client_web_socket.go

```diff
@@ -124,7 +123,7 @@ func (c *WebSocketClient) Start(ctx context.Context, executors []WebsocketExecut
 				}
 			}
 		case <-ctx.Done():
-			log.Println("interrupt")
+			logger.Println("interrupt")
```

Comment:
Since the logs are not set from the user's point of view, but from the developer's point of view, it seems okay to default to no log output 👍